cnxepub/scripts/collated_single_html/main.py

```diff
@@ -34,9 +34,12 @@ def main(argv=None):
     parser.add_argument('-d', '--dump-tree', action='store_true',
                         help='Print out parsed model tree.')
 
-    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+    parser.add_argument('-o', '--output', type=argparse.FileType('w+'),
```

Comment:
FYI, I believe the filetype allows for output direction. For example, `cat foo.zip | $script -i - -o - > bar.zip`, but this will fail because the python zipfile module can't use a streaming file buffer. Do to this I've tried not to use `argparse.FileType` for input/output operations.